statsd/pom.xml

```diff
@@ -130,8 +130,18 @@
           <artifactId>apacheds-protocol-ldap</artifactId>
           <groupId>org.apache.directory.server</groupId>
         </exclusion>
+        <exclusion>
+          <groupId>org.glassfish</groupId>
+          <artifactId>jakarta.el</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>jakarta.el</artifactId>
+      <scope>provided</scope>
+      <version>3.0.4</version>
```

Comment:
I would also mention that we are updating the version for both uaa and statsd in the commit message. Its strange that we have to update the version in so many places.

Reply:
spring boot starter web dependency is used in all modules, hence required version update at all places.